examples/arkode/CXX_parallel/ark_diffusion_reaction_p.cpp

```diff
@@ -1105,8 +1105,8 @@ static int SetupMRI(SUNContext ctx, UserData* udata, N_Vector y,
   if (check_flag((void *)*arkode_mem, "MRIStepCreate", 0)) return 1;
 
   // Set method coupling table (solve-decoupled implicit method)
-  MRIStepCoupling C = MRIStepCoupling_LoadTable(ARKODE_MRI_GARK_ESDIRK34a);
-  if (check_flag((void*)C, "MRIStepCoupling_LoadTable", 1)) return 1;
+  MRIStepCoupling C = MRIStepCoupling_LoadTableByName("ARKODE_MRI_GARK_ESDIRK34a");
+  if (check_flag((void*)C, "MRIStepCoupling_LoadTableByName", 1)) return 1;
```

Comment:
> Are there other example problems that still use the named constants (to ensure more complete coverage by our CI)?

Not within the CXX_parallel directory, but the C_serial tests still use the enum constants instead of strings.

Reply:
That should be sufficient; thanks for checking.